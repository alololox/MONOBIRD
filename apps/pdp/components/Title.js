import React from "react";
import styled from "styled-components";

const H1 = styled.h1`
  font-size: 24px;
  line-height: 1.25em;
  text-transform: uppercase;
  margin: 30px 0 15px 0;
  text-align: center;
`;

function Title(props) {
  return <H1>{props.name}</H1>;
}

export default Title;
