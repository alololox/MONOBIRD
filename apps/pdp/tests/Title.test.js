import React from 'react';
import renderer from 'react-test-renderer';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
 
Enzyme.configure({adapter : new Adapter() });

import Title from '../components/Title';
const title = 'Test Title';
let wrapped = shallow(<Title>{title}</Title>);
describe('Title', () => {
  it('should render the Title Component correctly', () => {   
    expect(wrapped).toMatchSnapshot();
  });
  it('renders the Titles children', () => { 
    expect(wrapped.find('h1').text()).toEqual(title);
  });
});