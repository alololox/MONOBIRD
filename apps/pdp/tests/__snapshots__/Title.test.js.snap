// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should render the Title Component correctly 1`] = `
<Title__H1>
  Test Title
</Title__H1>
`;
